index.js

```diff
@@ -34,7 +34,7 @@ const N_DIV_2 = new BN('7fffffffffffffffffffffffffffffff5d576e7357a4501ddfe92f46
  * @prop {Buffer} s EC recovery ID
  */
 module.exports = class Transaction {
-  constructor (data) {
+  constructor (data, chainId) {
```

Comment:
@kvhnuke sorry this was a typo on my side, I didn't mean `chain_id` as we usually keep camelcase for these, ie. `chainId`.

@wanderer @kumavis what should be the parameter name?